en_US/rule/backend_oracle.md

```diff
@@ -42,6 +42,8 @@ Fill in the resource configuration:
 
 ![image](./assets/rule-engine/oracle_action_3.png)
```

Comment:
Unfortunately, the Chinese fonts do not display correctly in my machine to allow me to take a new screenshot of the resource creation for the Chinese translation. :crying_cat_face:

Reply:
it looks ok.

Reply:
Oh, I should have commented this in the `zh_CN` file.  That one is still missing the "Service Name" field, and when I change to Chinese, my browser shows some squares instead of the correct characters :sweat:

Reply:
Yeah, I wonder how they got it so big :thinking:
You think the new one should be larger?